easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.0-gompi-2022a.eb

```diff
@@ -29,29 +31,17 @@ builddependencies = [
 ]
 
 dependencies = [
-    ('file', '5.43'),  # provides libmagic
-    ('HDF5', '1.10.8'),  # version 1.12.x has changes to API and is not compatible
+    ('HDF5', '1.12.2'),
     ('libxml2', '2.9.13'),
     ('bzip2', '1.0.8'),
 ]
 
-# add addtional libraries needed to statically link HDF5 from EB
```

Comment:
@boegel are these settings for static linking really no longer required?

Reply:
The patching of `konfigure.perl` is only relevant when using the provided `configure` script (which calls `konfigure.perl`), but we're bypassing that and calling `cmake` ourselves here - see also https://github.com/ncbi/ncbi-vdb/wiki/Building-and-Installing-from-Source